### ISSUE TITLE
Unable to Voice Enable a bot created using Composer

### ISSUE THREAD

**Issue by cvarun94** [MicrosoftDocs/bot-docs#2220]
[Hi, thanks for the documentation. I have created a weather_bot using BOT COMPOSER by referring the tutorial provided and am trying to voice enable it by adding direct speech line channel after I publish it to azure, but it is not working as the changes you have mentioned(to botcontroller.cs and startup.cs) when made gives error upon building and and fails to publish to azure. Can you let me know if a bot created using Composer can be voice enabled and if it requires these changes mentioned in the documentation and if yes pls specify the where and what  changes are required to be made.. Thank You]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7a0b8671-359e-6026-fd3a-39331cea73f3
* Version Independent ID: 29bab4ad-243a-fb0a-99ed-d8336a92f648
* Content: [Use Direct Line Speech in your bot in Bot Framework SDK - Bot Service](https://docs.microsoft.com/en-us/azure/bot-service/directline-speech-bot?view=azure-bot-service-4.0)
* Content Source: [articles/directline-speech-bot.md](https://github.com/MicrosoftDocs/bot-docs-pr/blob/live/articles/directline-speech-bot.md)
* Service: **bot-service**
* GitHub Login: @JonathanFingold
* Microsoft Alias: **iawilt**

**Comment by cvarun94** [comment]
Hi @emgrol! 
Is there any update on this issue?

Thank You

**Comment by emgrol** [comment]
Hi @cvarun94, sorry for the delay. Can you clarify what you mean by voice enabled? You can add Speech to Composer bots (see the [Speech article](https://docs.microsoft.com/en-us/composer/concept-speech?tabs=v2x)). If you're looking to connect to Direct Line Speech specifically, try following the [Connect to Direct Line Speech instructions](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-channel-connect-directlinespeech?view=azure-bot-service-4.0#add-the-direct-line-speech-channel).

**Comment by cvarun94** [comment]
Hi @emgrol 
By voice enable I mean that I should be able to provide voice as input to the azure bot and get voice as an output and to achieve this I have already tried the Direct Line Speech but it is still not working and I am getting lot of errors. Kindly see here -https://github.com/microsoft/BotFramework-Composer/issues/9329  - any help on this would be highly appreciated.

Procedure Followed - 

* I created a simple bot with LUIS authoring using the BotComposer
* Published the bot created to azure by creating necessary resources including Speech Service
* Added direct line speech channel and added the created speech resource to it
* Wen Sockets was already enabled, I manually enabled the Streaming point checkpoint.
* Then took the speech service key and region and entered it in the Windows Voice Assistant client to check
* Getting error

Kindly let me know if further clarification is required. I have tried multiple times and followed the procedure mentioned in the documentation but still getting error.

Thank You
